projects/container_bench/testing/prepare.py

```
@@ -31,49 +56,47 @@ def cleanup():
         logging.info("preparing docker desktop")
         docker_desktop = DockerDesktopMachine()
         docker_desktop.start()
         docker = ContainerEngine("docker")
         docker.cleanup()
 
+
+def prepare_podman_platform_darwin():
+    logging.info("preparing podman machine")
+    machine = PodmanMachine()
+    if machine.is_running():
+        raise RuntimeError("Podman machine is running.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)